Core/clim-basic/geometry/bounding-rectangle.lisp

```diff
@@ -31,7 +31,7 @@
 ;;; 4.1.2 Bounding Rectangle Convenience Functions
 
 (defmacro with-bounding-rectangle* ((min-x min-y max-x max-y) region &body body)
-  ;; What is the purpose of this macro; IHMO m.-v.-b. looks as nice as with-b.-.r. .
+  ;; What is the purpose of this macro; IMHO m.-v.-b. looks as nice as with-b.-.r. .
```

Comment:
The macro is in the spec, so the comment can be deleted, I think.

Reply:
deleted